documentation/modules/exploit/multi/http/tomcat_partial_put_deserialization.md

```diff
@@ -0,0 +1,46 @@
+## Vulnerable Application
+This module exploits a Java deserialization vulnerability in Apache Tomcat's session restoration functionality 
+that can be exploited with a partial PUT to place an attacker controlled deserialization payload in the work directory. 
+For the exploit to succeed, writes must be enabled for the default servlet, and `org.apache.catalina.session.PersistentManager` must be 
+configured to use `org.apache.catalina.session.FileStore`.
+
+## Testing
+Set up a vulnerable Tomcat instance using https://github.com/PaloAltoNetworks/Unit42-timely-threat-intel/blob/main/2025-03-14-Testing-CVE-2025-24813.md#establishing-a-vulnerable-tomcat-configuration
```

Comment:
Please put the instructions here instead of linking to them, as we don't know for how long the link will be valid :)